desc/geometry/core.py

```diff
@@ -142,3 +156,119 @@ def __repr__(self):
             + str(hex(id(self)))
             + " (name={}, grid={})".format(self.name, self.grid)
         )
+
+
+def compute_orientation(Rm1, Rp1, Zm1, Zp1):
```

Comment:
Are we sure the orientation is entirely determined from only the `m=1, n=0`? I think so, but want to double-check this. What if you have a surface where the `m=1` coefficients are small compared to higher `m` modes?